lms/envs/common.py

```diff
@@ -948,6 +948,16 @@
     # .. toggle_target_removal_date: 2021-10-01
     # .. toggle_tickets: 'https://openedx.atlassian.net/browse/MICROBA-1405'
     'ENABLE_V2_CERT_DISPLAY_SETTINGS': False,
+
+    # .. toggle_name: FEATURES['DISABLE_AUTOENROLL_AFTER_COURSE_END']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to disable auto enrollment of user to a course
+    # .. if user is registering after the course has ended
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2021-12-09
+    # .. toggle_tickets: 'https://github.com/edx/edx-platform/pull/29538'
+    'DISABLE_AUTOENROLL_AFTER_COURSE_END': False,
```

Comment:
Let's call it `DISABLE_AUTO_ENROLL_AFTER_COURSE_END`, to match the naming convention in the edx-platform.